traffic_ops/traffic_ops_golang/deliveryservice/request/requests.go

```diff
@@ -430,6 +430,20 @@ func (req *TODeliveryServiceRequest) Delete(db *sqlx.DB, user auth.CurrentUser)
 	return nil, tc.NoError
 }
 
+// ChangeLogMessage implements the api.ChangeLogger interface for a custom log message
+func (req *TODeliveryServiceRequest) ChangeLogMessage(action string, db *sqlx.DB) (string, error) {
```

Comment:
Why does ChangeLogMessage take a db handle as an argument? Will there be cases were constructing a struct's log message requires querying the db?

Reply:
possibly with foreign keys,  but the id probably would be enough..   we should discuss more..